dali/sasha/saxref.cpp

```diff
@@ -1001,13 +1001,75 @@ class CNewXRefManagerBase
         heartbeatTimer.updatePeriod();
     }
 
-    void addBranch(IPropertyTree *root,const char *name,IPropertyTree *branch)
+    void doSaveToDebugPlane(bool saveToDebugPlane, const char *name, StringBuffer &datastr)
+    {
+        if (!saveToDebugPlane)
```

Comment:
minor: with saveToDebugPlane as a member, I'd make this an assert, and add the condition to call doSaveToDebugPlane in addBranch.